app/src/main/java/xyz/hisname/fireflyiii/service/FireflyAccountAuthenticator.kt

```diff
@@ -65,25 +66,23 @@ class FireflyAccountAuthenticator(private val context: Context): AbstractAccount
 
     override fun getAccountRemovalAllowed(response: AccountAuthenticatorResponse, account: Account): Bundle {
         runBlocking(Dispatchers.IO){
-            val fireflyUserDatabase = FireflyUserDatabase.getInstance(context).fireflyUserDao()
-            fireflyUserDatabase.deleteCurrentUser()
```

Comment:
This is what was causing the `homeViewModel.deleteUnusedAccount()` to cause trouble